Update community resources for migration guides (#11376)
withastro/docs: src/content/docs/en/guides/cms/wordpress.mdx

<Grid>
  <Card title="Astro WordPress Starter" href="https://astro.build/themes/details/astro-wordpress-starter/" thumbnail="astro-wordpress-starter.png"/>
</Grid>

## Community Resources

<CardGrid>

  <LinkCard href="https://dev.to/bngmnn/leveraging-wordpress-as-a-headless-cms-for-your-astro-website-a-comprehensive-guide-a4d" title="Introduction to Astro + WordPress"/>

  <LinkCard title="Shattering Headless WordPress Build Times with Astro's Content Layer API" href="https://andrewkepson.com/blog/headless-wordpress/build-time-astro-content-layer-api/"/>

</CardGrid>

:::note[Have a resource to share?]

> If you found (or made!) a helpful video or blog post about using headless WordPress with Astro, edit this page and add it here!

Author: ```suggestion
If you found (or made!) a helpful video or blog post about using headless WordPress with Astro, [add it to this list](https://github.com/withastro/docs/edit/main/src/content/docs/en/guides/cms/wordpress.mdx)!
```

<Grid>
  <Card title="Astro WordPress Starter" href="https://astro.build/themes/details/astro-wordpress-starter/" thumbnail="astro-wordpress-starter.png"/>
</Grid>

## Community Resources

<CardGrid>

  <LinkCard href="https://dev.to/bngmnn/leveraging-wordpress-as-a-headless-cms-for-your-astro-website-a-comprehensive-guide-a4d" title="Introduction to Astro + WordPress"/>
  <LinkCard title="Astro + WPGraphQL for more secure WordPress sites" href="https://www.youtube.com/watch?v=fWxn-r83ygQ" />
  <LinkCard title="Shattering Headless WordPress Build Times with Astro's Content Layer API" href="https://andrewkepson.com/blog/headless-wordpress/build-time-astro-content-layer-api/"/>
  <LinkCard title="How to Set Up a Headless WordPress Site with Astro" href="https://dev.to/mathiasahlgren/how-to-set-up-a-headless-wordpress-site-with-astro-3a2h" />
  <LinkCard title="Build a static site with WordPress and Astro" href="https://kinsta.com/blog/wordpress-astro/" />

>   <LinkCard title="Leveraging WordPress as a Headless CMS for Your Astro Website: API Configuration & Data Fetching" href="https://medium.com/@bangemann.dev/configure-wordpress-rest-api-setup-data-fetching-4af5161095f6" />
>   <LinkCard title="WordPress Headless with Astro - Installing Astro and Fetching posts with WP-GraphQL" href="https://www.youtube.com/watch?v=2PSqABrME28" />
>   <LinkCard title="Going Headless WordPress with Astro" href="https://www.youtube.com/watch?v=MP2TR6Z_YTc" />

Reviewer: nit: better balance
```suggestion
  <LinkCard title="Going Headless WordPress with Astro" href="https://www.youtube.com/watch?v=MP2TR6Z_YTc" />
  <LinkCard title="Leveraging WordPress as a Headless CMS for Your Astro Website: API Configuration & Data Fetching" href="https://medium.com/@bangemann.dev/configure-wordpress-rest-api-setup-data-fetching-4af5161095f6" />
  <LinkCard title="WordPress Headless with Astro - Installing Astro and Fetching posts with WP-GraphQL" href="https://www.youtube.com/watch?v=2PSqABrME28" />
```
Here's the changes using `iPad` view in Firefox adaptive view:
|Before|After|
|---|---|
|![Preview of the links before changes](https://github.com/user-attachments/assets/e11ee5bf-0477-4535-b036-a86cc0543d71)|![Preview of the links after changes](https://github.com/user-attachments/assets/277fd9d2-1107-43b8-97a3-b8ed187f4390)|
